test/functional/cmdLineTests/jfr/playlist.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+Copyright IBM Corp. and others 2024
+
+This program and the accompanying materials are made available under
+the terms of the Eclipse Public License 2.0 which accompanies this
+distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+or the Apache License, Version 2.0 which accompanies this distribution and
+is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+This Source Code may also be made available under the following
+Secondary Licenses when the conditions for such availability set
+forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] https://openjdk.org/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../TKG/playlist.xsd">
+	<include>../variables.mk</include>
+	<test>
+		<testCaseName>cmdLineTester_jfr</testCaseName>
+		<variations>
+			<variation>NoOptions</variation>
+		</variations>
+		<command>
+			$(JAVA_COMMAND) $(CMDLINETESTER_JVM_OPTIONS) -Xdump \
+			-DEXE=$(SQ)$(JAVA_COMMAND) $(JVM_OPTIONS)$(SQ) \
+			-DMETADATA_BLOB_PATH=$(Q)$(TEST_RESROOT)$(D)metadata.blob$(Q) \
```

Comment:
Does this just establish where the JFR sample is written? If so, why do we need the `.blob` file in the repository? I don't see anything here that triggers reading that file.

Reply:
Is the same true for openjdk (i.e. that a metadata file is required to use JFR)? If not, is there a plan to remove that requirement for OpenJ9?

Reply:
How does one create the metafile in the first place?

Reply:
@tajila

Reply:
> Is the same true for openjdk (i.e. that a metadata file is required to use JFR)? If not, is there a plan to remove that requirement for OpenJ9?

Yes, openJDK needs it as well, however, they can generate it at build time. We are making steps to be able to do that as well, but we are not there yet.